config/7.11.0/common.yaml

```diff
@@ -680,6 +680,8 @@ servers:
                       value: "[[.Build.DisablePullDeps]]"
                     - name: KIE_SERVER_DISABLE_KC_VERIFICATION
                       value: "[[.Build.DisableKCVerification]]"
+                    - name: JBPM_CLUSTER
+                      value: "true"
```

Comment:
if this is set statically to true, how would the user change to false? maybe this needs to be a template var instead?
`[[.JbpmCluster]]` ?